packages/appstore/package.json

```diff
@@ -29,7 +29,6 @@
         "@deriv/cashier": "^1.0.0",
         "@deriv/components": "^1.0.0",
         "@deriv/cfd": "^1.0.0",
-        "@deriv/hooks": "^1.0.0",
```

Comment:
is this related to this card? or it's just a cleanup?

Reply:
@shayan-deriv Not related to this PR, But I saw there are 2 `@deriv/hooks` dependencies here so I just removed one of them 🙇🏻